baybe/surrogates/base.py

```diff
@@ -118,13 +144,18 @@ def _posterior(self, candidates: Tensor) -> tuple[Tensor, Tensor]:
             See :func:`baybe.surrogates.Surrogate.posterior`.
         """
 
-    def fit(self, searchspace: SearchSpace, train_x: Tensor, train_y: Tensor) -> None:
+    def fit(
+        self,
+        searchspace: SearchSpace,
+        objective: Objective,
+        measurements: pd.DataFrame,
+    ) -> None:
         """Train the surrogate model on the provided data.
 
         Args:
             searchspace: The search space in which experiments are conducted.
-            train_x: The training data points.
-            train_y: The training data labels.
+            objective: The objective to be optimized.
+            measurements: The training data in experimental representation.
```

Comment:
Probably not that important, but is there any way that we could verify that `measurements` is actually in experimental representation? I guess that is not necessary, since handing over measurements in computational representation would also just work? Or is this something we might need to take care of?

Reply:
easiest verify is whether all parameter names are in columns
parameters in comp_rep will have additional suffixes